README.md

```diff
@@ -208,44 +209,57 @@ beforeEach(() => {
 
   mockClient.setRequestHandler(
     SUBSCRIBE_DOG_DOCUMENT,
-    () => Promise.resolve(mockSubscription));
+    () => mockSubscription);
 
   wrapper = mount(
     <ApolloProvider client={mockClient}>
-      <Dog name="Rufus" />
+      <DogSubscription name="Rufus" />
     </ApolloProvider>
   );
 });
 
-it('renders the dog name and breed', () => {
-  mockSubscription.next({ data: { dog: { id: 1, name: 'Rufus', breed: 'Poodle' } } });
-  expect(wrapper.text()).toContain('Rufus is a Poodle');
+it('renders the initial dog details', () => {
+  act(() => {
+    mockSubscription.next({ data: { dog: { id: 1, name: 'Rufus', numberOfBarks: 0 } } });
+  });
+  expect(wrapper.text()).toContain('Rufus has barked 0 time(s)');
+});
+
+it('renders updated dog details when new data is received', () => {
+  act(() => {
+    mockSubscription.next({ data: { dog: { id: 1, name: 'Rufus', numberOfBarks: 1 } } });
+  });
+  expect(wrapper.text()).toContain('Rufus has barked 1 time(s)');
```

Comment:
Not all framework guarantee the order of tests, might be better if it was in one test with two expects.

Reply:
Good spot, thanks. I've updated the sample to be one test and two sets of assertions as suggested.